Gemfile

```diff
@@ -21,6 +21,13 @@ gem 'gibberish'
 gem 'hiredis'
 gem 'http_accept_language'
 gem 'identity-hostdata', github: '18F/identity-hostdata', tag: 'v0.4.1'
+require File.join(__dir__, 'app', 'services', 'lambda_jobs', 'git_ref.rb')
+gem 'identity-idp-functions',
+  require: 'identity/idp/functions',
+  github: '18F/identity-idp-functions',
+  # uncomment this for local dev
+  # path: '../identity-idp-functions',
```

Comment:
I had some problems with this...the particular SHA does exist...but isn't merged? Does Github over HTTPs not allow other branches maybe?

```
Fetching https://github.com/18F/identity-idp-functions.git
fatal: Could not parse object 'a5c1f89954bff1af1fdc92f37860aafd16afed87'.
Revision a5c1f89954bff1af1fdc92f37860aafd16afed87 does not exist in the repository https://github.com/18F/identity-idp-functions.git. Maybe you misspelled it?
```

Reply:
that is odd...it seems correct to me

Reply:
Also looks to me like it should work. 🤔 

A couple ideas:

- Prefixing ref with `refs/`. While the documentation doesn't mention anything about it, the [source code includes a condition to include this in extra refs when fetching the repository](https://github.com/rubygems/rubygems/blob/9b25442ea8e031dd57466bbf5c123fe182dfbd78/bundler/lib/bundler/source/git/git_proxy.rb#L91).
- Use the merge commit SHA `6369e830bd6d9594581e3046f396b57923432589` ([relevant documentation](https://developer.github.com/v3/pulls/#response-1))

Reply:
I tried both those, no dice. I'm hoping that once the PR is merged, I can use the commit -- the bummer is the `master` branch doesn't have a `.gemspec` yet so I can't use that SHA

Reply:
found a working SHA in e1e2573, committed it 🤷